UPGRADE-8.0.md

```diff
@@ -112,7 +112,7 @@ Remove the file `app/autoload.php`. It is no longer used by the system (see belo
 The functionality "annotation support" was removed. This file was calling a
 deprecated function `AnnotationRegistry::registerLoader()`. If needed annotations can still be configured and used
 directly in a project.
-However with php 8 you should use attributes instead.
+However with php > 8 you should use attributes instead.
```

Comment:
Technically it is php >= 8

Reply:
we require php 8.1 in the new version - so I think this should be ok